spec/schema.json

```diff
@@ -213,7 +243,7 @@
         "legal_form": {
           "type": "string"
         },
-        "law_governed": {
+        "governing_law": {
```

Comment:
Oh, just a thought on this - it will mean a change in the API DTO model class too. And in the chips-filing-consumer model that I'm currently working on. We'll have to be careful with these renames/changes on any objects that we've mapped in the API and filing consumer...

Reply:
ok, i'm happy to change it back to law_governed

Reply:
I don't mind the change, just need to be sure not to forget to change this elsewhere.

Reply:
If you change it we need to change on line 233